pkg/storage/couchdb/couchdbstore.go

```diff
@@ -64,6 +64,11 @@ func NewProvider(hostURL string, opts ...Option) (*Provider, error) {
 
 	p := &Provider{hostURL: hostURL, couchDBClient: client, dbs: map[string]*CouchDBStore{}}
 
+	_, err = client.Ping(context.Background())
```

Comment:
please fix the unit tests as well... there's at least one where no error is expected after providing a `wrongURL`.

Reply:
done